el8/config.yaml

```diff
@@ -31,6 +31,13 @@ groups:
   runtime:
     docker: Dockerfile.${group0}
     from: library/almalinux:8
+    groups:
+      x86_64:
+        tags:
+          ${group1}-${group0}:
+  buildtime:
+    docker: Dockerfile.${group0}
+    from: ${container}:${group1}-runtime
```

Comment:
@aandvalenzuela , should we keep the `SKIP_TESTS: true` under `runtime` container?

Reply:
I was setting it to true to trigger the testing in Jenkins, should I test it manually instead?